.commitlintrc.yml

```diff
@@ -1,2 +1,17 @@
 extends:
   - '@commitlint/config-conventional'
+rules:
+  type-enum: [2, 'always', [
+    'build',
+    'docs',
+    'feat',
+    'fix',
+    'perf',
+    'refactor',
+    'revert',
+    'test'
+  ]]
+  scope-enum: [2, 'always', [
+    'spectral'
+  ]]
+  scope-empty: [0, 'never']
```

Comment:
I think the `scope-empty` rule can be omitted as it's the same as the default value, unless you prefer it being explicit?